src/lib/keyboard.ts

```diff
@@ -1,10 +1,12 @@
 import { useEffect } from 'react'
 import { osName } from './utils'
+import isElectron from 'is-electron'
 
 export const useGlobalKeyDownHandler = (
   handler: (event: KeyboardEvent) => void
 ) => {
   return useEffect(() => {
+    if (!isElectron()) return
```

Comment:
This looks just wrong
It is same to 
`return window.addEventListener('keydown', handler)`
If you want to return `void`, you should do `{ return }` not just `return` to prevent these problems.

Also, what are we going to do shortcuts for electron app?